src/mainwindow.cpp

```diff
@@ -89,6 +89,8 @@ MainWindow::MainWindow(lib::settings &settings, lib::paths &paths,
 		new MainMenuBar(spotify, settings, httpClient, cache, this);
 	}
 
+	// Queue dock widget is initially hidden and can be toggled via the toolbar button
```

Comment:
Comments like these exist in a couple of places that provide no additional information that the code itself already doesn't, comments are for describing *why*, not *what*.